### PR TITLE
Migrate to app dir

### PR DESCRIPTION
**Pull request by Moltenship**

### CODE REVIEW

**Review by moetazaneta — APPROVED** [review]

File: src/app/auth/login/page.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { LoginPage as Login } from 'src/features/auth/pages/LoginPage/LoginPage';
+
+const LoginPage = async() => (
+  <Login />
+);
```
Comment (NightOfNights) [inline src/app/auth/login/page.tsx]: Why do you need `async` here? I guess it would work without it

File: src/app/layout.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+  const atomValues = { settings, isMobile };
+
+  const session = await getServerSession(authOptions);
+
+  return (
+    <html lang="en">
+      <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      </head>
```
Comment (NightOfNights) [inline src/app/layout.tsx]: As far as I know, it's not need. Next.js adds it automatically

File: src/components/Input/Input.tsx [hunk 3]
```diff
@@ -1,5 +1,7 @@
+'use client';
+
```
Comment (NightOfNights) [inline src/components/Input/Input.tsx:2]: Please, double check if `use client` is really needed here. This stays true for other general components

File: src/components/Link/Link.tsx [hunk 4]
```diff
@@ -76,7 +76,10 @@ const LinkComponent: FC<PropsWithChildrenAndClass<Props>> = (
   }
 
   return (
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
     <NextLink
+      prefetch={false}
```
Comment (NightOfNights) [inline src/components/Link/Link.tsx]: Why it's needed? Let's provide docs

**Review by NightOfNights — APPROVED** [review]

**Pull request merged**